JPS_VIRTUALSENSOR/AermodAgent/src/main/java/com/cmclinnovations/aermod/AermodAgent.java

```diff
@@ -239,10 +255,43 @@ public void processRequestParameters(DerivationInputs derivationInputs, Derivati
         // then create a layer with that style as default
         GeoServerVectorSettings geoServerVectorSettings = new GeoServerVectorSettings();
         geoServerVectorSettings.setDefaultStyle("dispersion_style");
-        for (int i = 0; i < dispLayerNames.size(); i++) {
-            geoServerClient.createPostGISLayer(EnvConfig.GEOSERVER_WORKSPACE, EnvConfig.DATABASE, dispLayerNames.get(i),
-                    geoServerVectorSettings);
+        // Set GeoServer layer names. Include simulationDirectory in the name to prevent
+        // name collision.
+        List<String> dispLayerNames = new ArrayList<>();
+        for (int i = 0; i < receptorHeights.size(); i++) {
+            double height = receptorHeights.get(i);
+            int receptorHeightInt = (int) Math.round(height);
```

Comment:
what if we have heights like 1.2 & 1.4 and both round to 1?

Reply:
Not sure if it is possible to have decimal points in the GeoServer layer name. Maybe I could do something like 

String receptorHeight = String.valueOf(height).replace(".","_dp_");